lib/store/src/GlobalsStore.ts

```diff
@@ -5,6 +5,7 @@ import type { Globals, GlobalTypes } from '@storybook/csf';
 import { deepDiff, DEEPLY_EQUAL } from './args';
 import { getValuesFromArgTypes } from './csf/getValuesFromArgTypes';
 
+// TODO removing this behavior breaks test, so more work is required to fix it
```

Comment:
@tmeasday let's chat about this, how to remove.

Reply:
We'll need to actually change the behaviour here and add tests for it. Perhaps we can make a ticket for this one?